crud/common/sharding.lua

```diff
@@ -2,12 +2,24 @@ local vshard = require('vshard')
 local errors = require('errors')
 
 local BucketIDError = errors.new_class("BucketIDError", {capture_stack = false})
+local GetReplicasetsError = errors.new_class('GetReplicasetsError')
 
 local utils = require('crud.common.utils')
 local sharding_key_module = require('crud.common.sharding_key')
 
 local sharding = {}
 
+function sharding.get_replicasets_by_sharding_key(bucket_id)
```

Comment:
May be `get_replicasets_by_bucket_id()?`

Reply:
Renamed `get_replicasets_by_sharding_key` -> `get_replicasets_by_bucket_id`.